pkg/generator/generator.go

```diff
@@ -215,14 +223,15 @@ func (g Generator) GenerateConfig(ctx context.Context, installation, app, namesp
 		Name:      name,
 		Namespace: namespace,
 		Labels: map[string]string{
-			key.KubernetesManagedByLabel: "Helm",
-			key.GiantswarmManagedByLabel: project.Name(),
-			key.ConfigVersion:            ref,
+			key.KubernetesManagedByLabel:  "Helm",
+			label.AppKubernetesName:       project.Name(),
```

Comment:
Shouldn't the value here set to the name of the configured app?

How does this